src/sentry_random.c

```diff
@@ -54,19 +54,20 @@ getrandom_devurandom(void *dst, size_t bytes)
 #ifdef SENTRY_PLATFORM_WINDOWS
 typedef BOOLEAN(WINAPI *sRtlGenRandom)(PVOID Buffer, ULONG BufferLength);
 
-static sRtlGenRandom pRtlGenRandom;
-
-SENTRY_CTOR (init_winapi) {
```

Comment:
I think this was the only usage of `SENTRY_CTOR`, was it? In which case would be good to also remove it.

Reply:
Yeah good idea, removed